core/indexer/elasticsearchDatabase_test.go

```diff
@@ -601,32 +602,36 @@ func TestDoBulkRequestLimit(t *testing.T) {
 	esDatabase, _ := newElasticSearchDatabase(args)
 
 	//Generate transaction and hashes
-	numTransactions := 1000
-	dataSize := 12345
-	txs, hashes := generateTransactions(numTransactions, dataSize)
 
-	header := &dataBlock.Header{}
-	txsPool := make(map[string]data.TransactionHandler)
-	for i := 0; i < numTransactions; i++ {
-		txsPool[hashes[i]] = &txs[i]
-	}
+	for i := 0; i < 1000; i++ {
+		numTransactions := 1
+		dataSize := 900001
+		txs, hashes := generateTransactions(numTransactions, dataSize)
 
-	miniblock := &dataBlock.MiniBlock{
-		TxHashes: make([][]byte, numTransactions),
-		Type:     dataBlock.TxBlock,
-	}
-	for i := 0; i < numTransactions; i++ {
-		miniblock.TxHashes[i] = []byte(hashes[i])
-	}
+		header := &dataBlock.Header{}
+		txsPool := make(map[string]data.TransactionHandler)
+		for i := 0; i < numTransactions; i++ {
```

Comment:
Use j instead i. The outer for uses also i for iteration

Reply:
done.